index.bs

```diff
@@ -342,6 +347,24 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
 
 NOTE: [[!AV1]] supports encoding either 3-component images (whose semantics are given by the <code>matrix_coefficients</code> element), or 1-component images (monochrome). When an image requires a different number of components, multiple auxiliary images may be used, each providing additional component(s), according to the semantics of their <code>aux_type</code> field. In such case, the maximum number of components is restricted by number of possible items in a file, coded on 16 or 32 bits.
 
+<h3 id="derived-images">Derived Image Items</h3>
+
+<h4 id="tonemap-derivation">Tonemap Derived Image Item</h4>
```

Comment:
Nit: Tonemap => Tone map

It is spelled as two words in [ISO/IEC DIS 23008-12/DAmd 1](https://www.iso.org/standard/89044.html).

Reply:
In HEIF it's spelled inconsistently as "tone map" or "tone-map" but more often as the latter (with the hyphen), at least in the version of the spec that I have

Reply:
Fixed